src/main/java/com/github/onsdigital/logging/builder/LogMessageBuilder.java

```diff
@@ -58,15 +44,15 @@ public LogMessageBuilder(Throwable t, Level level, String description) {
 
     public LogMessageBuilder(Throwable t, String description) {
         this.description = description;
-        this.logLevel = Level.ERROR;
-        this.parameters = new LogParameters();
+        logLevel = Level.ERROR;
```

Comment:
If we are moving towards removing use of `this` it may be worth considering changing the rest of the file as well to be more consistent.

Reply:
You need to use it in this case as the method variable and the class member have the same name. Adding `this` to the class memeber allows the compiler to differentiate them.

Reply:
I've updated it so now assignments to the class var use `this`, referencing a variable to get / update will just use the var name.